package.json

```
@@ -143,7 +143,7 @@
   "bundlesize": [
     {
       "path": "./dist/instantsearch.production.min.js",
-      "maxSize": "65.5 kB"
+      "maxSize": "65.65 kB"
```

We're used to increment with a 0.25KB step.

```suggestion
      "maxSize": "65.75 kB"
```